alpa/global_env.py

```diff
@@ -108,6 +108,7 @@ def __init__(self):
         self.use_scatter_gather = True
         self.eagerly_create_communicators = True
         self.use_memzero_for_gradient_accumulation = False
+        self.resharding_mode = "default"  # or "broadcast"
```

Comment:
Rename: `default`- > `send_recv`.
Later, we may use `broadcast` as the default

Reply:
OK